pyproject.toml

```diff
@@ -0,0 +1,15 @@
+[tool.poetry]
```

Comment:
This file is here because I created my branch from main...I think it makes sense to merge this file into develop too